tests/api/presenters/test_list_active_plans_presenter.py

```diff
@@ -6,16 +6,75 @@
     JsonInteger,
     JsonString,
 )
-from arbeitszeit_web.api_presenters.plans import ActivePlansPresenter
+from arbeitszeit_web.api_presenters.query_plans_api_presenter import (
+    QueryPlansApiPresenter,
+)
 from tests.api.presenters.base_test_case import BaseTestCase
 from tests.presenters.data_generators import QueriedPlanGenerator
 
 
-class TestGetPresenter(BaseTestCase):
+class TestViewModelCreation(BaseTestCase):
     def setUp(self) -> None:
         super().setUp()
+        self.presenter = self.injector.get(QueryPlansApiPresenter)
         self.queried_plan_generator = self.injector.get(QueriedPlanGenerator)
-        self.presenter = ActivePlansPresenter()
+
+    def test_presenter_raises_when_no_limit_was_requested(self):
+        use_case_response = self.queried_plan_generator.get_response(
+            requested_limit=None, requested_offset=10
+        )
+        with self.assertRaises(AssertionError):
+            self.presenter.create_view_model(use_case_response)
+
+    def test_presenter_raises_when_no_offset_was_requested(self):
+        use_case_response = self.queried_plan_generator.get_response(
+            requested_offset=None, requested_limit=10
+        )
+        with self.assertRaises(AssertionError):
```

Comment:
I think that it is generally a bad idea to use `AssertianError`s a means to handle errors. Assertions in the production code should be reserved for configurations that are meant to be impossible, for example an active plan without a activation date. The reason for this is that the python interpreter has an "optimized" mode, that can be started via `python -O`. Assertions are completely ignored in this mode.

https://stackoverflow.com/questions/2055557/what-is-the-use-of-the-o-flag-for-running-python